packages/docusaurus-theme-classic/codeTranslations/fr.json

```diff
@@ -29,7 +29,7 @@
   "theme.blog.post.plurals": "Un article|{count} articles",
   "theme.blog.post.readMore": "Lire plus",
   "theme.blog.post.readingTime.plurals": "Une minute de lecture|{readingTime} minutes de lecture",
-  "theme.blog.tagTitle": "{nPosts} taggés avec \"{tagName}\"",
+  "theme.blog.tagTitle": "{nPosts} tagués avec « {tagName} »",
```

Comment:
not easy to translate the "tag" concept in french. Not sure "taguer" verb is the best as imho it's more relevant for street art, but tagger is not great either 🤔 will merge this but if we have a better translation later we may revisit

Reply:
will note to add plurals syntax for this title

Reply:
Thanks for noting the plural agreement issue :)
taggés -> tagués is also a more adaptive spelling that reflects the pronunciation if I'm not mistaken.